language/stdlib/modules/libra_writeset_manager.move

```diff
@@ -9,7 +9,6 @@ module LibraWriteSetManager {
     use 0x0::LibraConfig;
```

Comment:
This module is so small that it might be worth combining with `LibraAccount` to avoid the funkiness of `bump_sequence_number`....

Reply:
My main concern is that this module will depend on `LibraConfig`. If we combine this with `LibraAccount` it might make our dependency story to be really weird.